### PR TITLE
Adding Pythagoras Theorem

### PR DESCRIPTION
**Pull request by cwh0430**
To calculate the hypothenuse, adjacent and base of the triangle

[Open in Gitpod](https://gitpod.io/from-referrer/)&nbsp;[know more](https://www.gitpod.io/docs/pull-requests/)

### Describe your change:

- [✓ ] Add an algorithm?
- [ ] Fix a bug or typo in an existing algorithm?
- [ ] Documentation change?

### Checklist:

- [✓ ] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Javascript/blob/master/CONTRIBUTING.md).
- [✓ ] This pull request is all my own work -- I have not plagiarized.
- [✓ ] I know that pull requests will not be merged if they fail the automated tests.
- [✓ ] This PR only changes one algorithm file. To ease review, please open separate PRs for separate algorithms.
- [✓ ] All new JavaScript files are placed inside an existing directory.
- [ ✓] All filenames should use the UpperCamelCase (PascalCase) style. There should be no spaces in filenames.
      **Example:**`UserProfile.js` is allowed but `userprofile.js`,`Userprofile.js`,`user-Profile.js`,`userProfile.js` are not
- [✓ ] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
- [✓ ] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

**Review by raklaptudirm — CHANGES_REQUESTED** [review]
Add some tests for the functions.

**Review by appgurueu — CHANGES_REQUESTED** [review]
Please rename the file to `PythagoreanTheorem`.

`calcAdjacent` and `calcBase` are practically the same function (`a = sqrt(c² - b²)` and `b = sqrt(c² - a²)`) - deduplicate them. Order shouldn't matter for `a` and `b` (adjacent vs. base).

**Review by appgurueu — CHANGES_REQUESTED** [review]
You've got two tests both for `adjacent` & `base` calculation. Please merge them in a single function.

File: Maths/PythagoreanTheorem.js [hunk 1]
```diff
@@ -1,35 +1,34 @@
...
+ * @see [calcOtherSides](https://en.wikipedia.org/wiki/Pythagorean_theorem)
+ * @example calcOtherSides(4, 3) = 2
  */
 const calcOtherSides = (hypothenuse, side1) => {
   if (side1 >= hypothenuse) {
     return 'Length of side1 must be smaller than hypothenuse'
   }
-  const side2 = Math.sqrt((hypothenuse ** 2) - (side1 ** 2))
-  return side2
+    const side2 = Math.sqrt((hypothenuse ** 2) - (side1 ** 2))
```
Comment (appgurueu) [inline Maths/PythagoreanTheorem.js:27]: indentation here is off

**Review by cwh0430 — COMMENTED** [review]
added test functions

**Review by appgurueu — APPROVED** [review]
Looks good enough (would perhaps rename `calcOtherSides` to `calcOtherSide` since it calculates *one* other side).

**Review by appgurueu — DISMISSED** [review]

File: Maths/PythagoreanTheorem.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  if (side1 >= hypothenuse) {
+    return 'Length of side1 must be smaller than hypothenuse'
+  }
+  const side2 = Math.sqrt((hypothenuse ** 2) - (side1 ** 2))
+  return side2
+}
+
+export {
+  calcHypothenuse,
+  calcOtherSides,
```
Comment (appgurueu) [inline Maths/PythagoreanTheorem.js:33]: You forgot to rename this too.

**Review by appgurueu — APPROVED** [review]

File: Maths/test/PythagoreanTheorem.test.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
+import * as pytheorem from "../PythagoreanTheorem"
+
+test('Testing on hypothenuse calculation', () => {
+  const hypothenuse = pytheorem.calcHypothenuse(2.0, 3.0)
```
Comment (appgurueu) [inline Maths/test/PythagoreanTheorem.test.js:4]: 4 + 9 = 13; sqrt(13) != 4

File: Maths/test/PythagoreanTheorem.test.js [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+  expect(hypothenuse).toBe(4.0)
+})
+
+test('Testing on other sides calculation', () => {
+  const side = pytheorem.calcOtherSide(4.0, 3.0)
```
Comment (appgurueu) [inline Maths/test/PythagoreanTheorem.test.js:9]: 16 - 9 = 7; sqrt(7) != 2

**Review by appgurueu — CHANGES_REQUESTED** [review]
Please use pythagorean triplets as tests. Your current tests are invalid.

**Pull request closed without merging**